internal/store/pg/migrations/1_initial.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE users(id varchar(36) PRIMARY KEY, user_name varchar(255), hash_password varchar(255), deleted boolean DEFAULT FALSE, registration_date timestamp);
```

Comment:
Could you please format this code?
https://www.dbvis.com/thetable/sql-formatter-definition-and-best-tools/#:~:text=An%20SQL%20formatter%20is%20a,or%20plugin%20in%20your%20projects.